crates/libs/windows/src/core/unknown.rs

```diff
@@ -7,6 +7,18 @@ use super::*;
 #[repr(transparent)]
 pub struct IUnknown(core::ptr::NonNull<core::ffi::c_void>);
 
+impl IUnknown {
+    /// Creates an `IUnknown` value by taking ownership of the `raw` pointer.
+    pub unsafe fn from_raw_owned(raw: *mut core::ffi::c_void) -> Self {
```

Comment:
You might consider calling this just `from_raw` as usually the assumption is that `from` functions return owned values. While this doesn't have symmetry with the `from_raw_borrowed` function it is symmetrical with other `from_raw` functions for other types.